<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testStackedWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>800</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="ButtonRegister">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>540</y>
         <width>250</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
background-color: rgba(234, 252, 185, 180);
border-radius:20px;
}

QPushButton:hover{
border-radius:20px;
	
	background-color: rgb(217, 238, 112);
}</string>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelReTitle">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>291</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background:transparent;
	font: 57 50pt &quot;Ubuntu&quot;;
</string>
       </property>
       <property name="text">
        <string>Resgiter</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelReBackground">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>700</width>
         <height>800</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>800</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>800</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/img/Cover.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditUserName">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>220</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background:transparent;
border:none;
border-bottom:2px solid rgb(46, 52, 54,180);
	font: 57 32pt &quot;Ubuntu&quot;;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>UserName</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditSetPs">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>320</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background:transparent;
border:none;
border-bottom:2px solid rgb(46, 52, 54,180);
	font: 57 32pt &quot;Ubuntu&quot;;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditConfirmPs">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>420</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background:transparent;
border:none;
border-bottom:2px solid rgb(46, 52, 54,180);
	font: 57 32pt &quot;Ubuntu&quot;;
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonToLogin">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>750</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
background-color: rgba(234, 252, 185, 180);
border-radius:20px;
}

QPushButton:hover{
border-radius:20px;
	
	background-color: rgb(217, 238, 112);
}</string>
       </property>
       <property name="text">
        <string>返回登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonRegsiterClose">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>0</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
background-color: rgba(234, 252, 185, 180);
border-radius:2px;
}

QPushButton:hover{
border-radius:2px;
	
	background-color: rgb(217, 238, 112);
}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <zorder>labelReBackground</zorder>
      <zorder>ButtonRegister</zorder>
      <zorder>labelReTitle</zorder>
      <zorder>lineEditUserName</zorder>
      <zorder>lineEditSetPs</zorder>
      <zorder>lineEditConfirmPs</zorder>
      <zorder>ButtonToLogin</zorder>
      <zorder>ButtonRegsiterClose</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>801</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:50px</string>
       </property>
       <widget class="QLabel" name="labelbackground">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>700</width>
          <height>800</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>800</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>800</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelbackground{
border-image: url(:/new/prefix1/img/Cover.jpg);
border-radius:50px ;
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="img.qrc">:/new/prefix1/img/Cover.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonLogin">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>510</y>
          <width>250</width>
          <height>60</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{
background-color: rgba(234, 252, 185, 180);
border-radius:20px;
}

QPushButton:hover{
border-radius:20px;
	
	background-color: rgb(217, 238, 112);
}</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelLoTitle">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>90</y>
          <width>241</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background:transparent;
	font: 57 50pt &quot;Ubuntu&quot;;
</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditUser">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>280</y>
          <width>311</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
background:transparent;
border:none;
border-bottom:2px solid rgb(46, 52, 54,180);
	font: 57 32pt &quot;Ubuntu&quot;;
}
	</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Accounts</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>400</y>
          <width>311</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
background:transparent;
border:none;
border-bottom:2px solid rgb(46, 52, 54,180);
	font: 57 32pt &quot;Ubuntu&quot;;
}
	</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonToRegister">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>610</y>
          <width>250</width>
          <height>60</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgba(234, 252, 185, 180);
border-radius:20px;
}

QPushButton:hover{
border-radius:20px;
	
	background-color: rgb(217, 238, 112);
}</string>
        </property>
        <property name="text">
         <string>没有账号？去注册</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonLoginClose">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{
background-color: rgba(234, 252, 185, 180);
border-radius:20px;
}

QPushButton:hover{
border-radius:20px;
	
	background-color: rgb(217, 238, 112);
}</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditRead">
        <property name="geometry">
         <rect>
          <x>463</x>
          <y>570</y>
          <width>221</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
